src/languages/1c.js

```diff
@@ -443,6 +443,12 @@ export default function(hljs) {
     ]
   };
 
+  const PUNCTUATION = {
+    match: /[;|(|)|+|\-|:|=|,]/,
```

Comment:
What are all the `|` for... `|` is just another character [with no special meaning] inside a `[]` block... I think you want to just remove them all?  Or have it only a single time if `|` itself is also punctuation...

Reply:
> What are all the `|` for... `|` is just another character [with no special meaning] inside a `[]` block... I think you want to just remove them all? Or have it only a single time if `|` itself is also punctuation...

You're right Josh. This is a mistake. Deleted it. Thanks!